packages/app/app/components/LibraryView/LibraryAlbumGrid/index.js

```diff
@@ -23,7 +23,7 @@ export default compose(
           .thru(_.head)
           .value(),
         thumb: _(group)
-          .map('image[0][\'#text\']')
```

Comment:
Don't change this, I'm unifying and overhauling search and this will be replaced anyway.

Reply:
ok

Reply:
actually this component will only be used with local files, its way more easier to manage a thumbnail property than an array with a single object with a #text property.